maxim/logger/livekit/realtime_session.py

```
@@ -0,0 +1,236 @@
+)
+from .store import get_maxim_logger, get_session_store
+
+
+def intercept_realtime_session_emit(self: RealtimeSession, *args, **kwargs):
+    """
+    This function is called when the realtime session emits an event.
+    """
+    event = args[0]
+    if event == "openai_client_event_queued":
+        # Here we are buffering the session level audio buffer first
+        type = args[1].get("type")
```

Pylint (3.3.7)

[warning] Redefining built-in 'type'

(W0622)